src/runtime_src/core/tools/xbmgmt2/OO_Config.cpp

```diff
@@ -142,37 +142,33 @@ show_device_conf(xrt_core::device* device)
   }
 
   try {
-    std::cout << "\tSecurity level: ";
     auto sec_level = xrt_core::device_query<xrt_core::query::sec_level>(device);
-    std::cout << sec_level << "\n";
+    std::cout << boost::format("  %-33s: %s\n") % "Security level" % sec_level;
   }
   catch (const std::exception& ex) {
     std::cout << ex.what() << "\n";
   }
 
   try {
-    std::cout << "\tRuntime clock scaling enabled status: ";
     auto scaling_enabled = xrt_core::device_query<xrt_core::query::xmc_scaling_enabled>(device);
-    std::cout << scaling_enabled << "\n";
+    std::cout << boost::format("  %-33s: %s\n") % "Runtime clock scaling enabled" % scaling_enabled;
   }
   catch (const std::exception& ex) {
-    std::cout << ex.what() << "\n";
+    //safe to ignore. These sysfs nodes are not present for u30
```

Comment:
Hmm...this comment suggests that the code is U30 aware.  

Is this safe to ignore for ALL cards?

Reply:
@maxzhen, can you chime in on this? Is there a scenario where the card requires "xmc_scaling_enabled" sysfs node but the driver fails to populate it? Is it safe to ignore this catch?

Reply:
in that case, it's a driver bug and should be fixed by driver. you should only ignore the case when you don't find this sysfs node. if you find it but can't do anything IO on it, then that is an error that you should not ignore.

Reply:
If I understand correctly, if the catch is tripped is because of a driver bug.   So in this case, we should at least report the exception.

Reply:
can we differentiate if the error is because the sysfs node is not found or something else? it is not a bug if the node is not found and we should ignore it.

Reply:
No, we only throw one kind of exception. We can't differentiate it

Reply:
Then we have to ignore it.

Reply:
Bummer :^(